### ISSUE TITLE
Dispute: appeal state is confusing

### ISSUE THREAD

**Issue by sohkai** [aragon/court-dashboard#332]
From

<img width="1121" alt="Screen Shot 2020-05-19 at 11 01 21 AM" src="https://user-images.githubusercontent.com/4166642/82307053-231b8c80-99c0-11ea-8ea1-3ad926f7f1c7.png">

It is not really clear which appeal outcome has been proposed, and thereby which "confirm appeal" options are available (without going into the panel).

Perhaps we need to change the language around "confirm appeal", or we can update the "outcome" text in the timeline to be "appeal outcome"?

**Comment by john-light** [comment]
+1 to changing "Outcome" to "Appeal outcome", just as the "Vote revealed" section below it says "Jury outcome".

Also +1 to changing the language around "confirm appeal", as I have suggested changing it from "confirm appeal" to "challenge appeal" to make it clear that performing this action means you disagree with the appealing party.

**Comment by fabriziovigevani** [comment]
Yes also think `challenge appeal` makes more sense.

> +1 to changing "Outcome" to "Appeal outcome", just as the "Vote revealed" section below it says "Jury outcome".

In the case a juror challenges the appeal, what do you think would be a reasonable outcome text `Challenge appeal outcome`  don't think would fit

**Comment by sohkai** [comment]
Maybe just "Appeal challenged"? At that point, it doesn't really matter for jurors / others which side either appealer took—we're just on to another round.

**Comment by john-light** [comment]
> In the case a juror challenges the appeal

I think it doesn't matter whether it's a juror that appeals/ challenges or not, since the effect is the same.

**Comment by fabriziovigevani** [comment]
> it doesn't really matter for jurors / others which side either appealer took—we're just on to another round.

You mean to not show the outcome chosen at all? 

I think it would still valuable information for the appealers to compare their outcome chosen with the final ruling in order to know if they will receive appeal rewards or not.

> Challenge outcome is another option, to keep it consistent with the _____ outcome format.

I like `Challenge outcome`

**Comment by sohkai** [comment]
> I think it would still valuable information for the appealers to compare their outcome chosen with the final ruling in order to know if they will receive appeal rewards or not.

This is true, but I wonder if the timeline is the best place for this information. Only the appealer's account really cares about how they appealed, and it might be something to put in the primary detail slot instead?

**Comment by john-light** [comment]
> Only the appealer's account really cares about how they appealed, and it might be something to put in the primary detail slot instead?

Anyone who wants to challenge the appealing party also cares about how they appealed.
